packages/react-native/scripts/react_native_pods.rb

```diff
@@ -439,6 +449,10 @@ def react_native_post_install(
   NewArchitectureHelper.set_clang_cxx_language_standard_if_needed(installer)
   NewArchitectureHelper.modify_flags_for_new_architecture(installer, NewArchitectureHelper.new_arch_enabled)
 
+  if ENV['USE_HERMES'] == '0'
```

Comment:
```suggestion
  if ENV['USE_HERMES'] == '0' && ENV['USE_THIRD_PARTY_JSC'] != '1'
```

that would save the warning if people use the 3rd party jsc already